sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Hook/AlertingHook.cs

```diff
@@ -8,6 +8,8 @@
 namespace Azure.AI.MetricsAdvisor.Models
 {
     /// <summary>
+    /// An alert notification to be triggered After an anomaly is detected by Metrics Advisor.
+    /// An alert setting can be used with multiple detection configurations, various parameters are available to customize your alert rule.
     /// </summary>
     [CodeGenModel("HookInfo")]
     public partial class AlertingHook
```

Comment:
We also have a `Description` and `ExternalLink` properties in the generated part of the code.